src/main/routes/receiver.ts

```diff
@@ -1,49 +1,185 @@
 import * as express from 'express'
-import * as config from 'config'
 
-import { Paths as AppPaths } from 'app/paths'
+import { Paths, Paths as AppPaths } from 'app/paths'
```

Comment:
Why would both `Paths` and `AppPaths` be needed if they reference the same object?

Reply:
they aren't